Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class Credentials
+    {
+        private const string Separator = ",";
+
+        public string CredentialsFile { get; } = "credentials.db";
+
+        public Credentials()
+        {
+        }
+
+        public Credentials(string credentialsFile)
+        {
+            CredentialsFile = credentialsFile ?? throw new ArgumentNullException(nameof(credentialsFile));
+        }
+
+        private bool FindUser(User user)
+        {
+            string line;
+            bool found = false;
+
+            StreamReader reader;
+            try
+            {
+                reader = new StreamReader(CredentialsFile);
+                while ((line = reader.ReadLine()) != null)
+                {
+                    var isValid = TryParse(line, out User credentials);
```

Comment:
can inline this if you want, no need for the variable

Reply:
Indeed, I updated it.